avocado/core/loader.py

```diff
@@ -44,6 +44,41 @@
 ALL = True
 
 
+def filter_test_tags(test_suite, filter_by_tags):
+    '''
+    The filtering mechanism is limited to instrumented tests with tags set
+    '''
```

Comment:
I can certainly switch styles, but the same way you can comment out `"""` with `'''`, you can do the opposite.